src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -47,6 +44,11 @@ private static void setupCommandTests(Application application) {
 
 		application.addTest("Run command", new SelectRunCurrentCommandOptionListener(DriverFeature.getDriverManager()));
 
+		application.addTest("⬤ | Record commands", new MacroToggleListener(DriverFeature.getDriverManager()));
+
+		application.addTest("✖ | Clear recorded", new MacroClearListener(DriverFeature.getDriverManager()));
+
+		application.addTest("▶ | Run recorded", new MacroRecordListener(DriverFeature.getDriverManager()));
```

Comment:
Instead of running the command it would be more reusable to "load" it (as a current command). Then it could be executed using Run command option.